programmers/완주하지-못한-선수.js

```diff
@@ -0,0 +1,17 @@
+function solution(participants, completions) {
+  let count = {}
+  let answer = ``
```

Comment:
변수가 들어가지 않는다면 '', ""가 적절합니다
```suggestion
  let answer = '';
```